packages/r/raylib/xmake.lua

```diff
@@ -55,11 +57,21 @@ package("raylib")
     end)
 
     on_test(function (package)
-        assert(package:check_cxxsnippets({test = [[
-            void test() {
-                InitWindow(100, 100, "hello world!");
-                Camera camera = { 0 };
-                UpdateCamera(&camera);
-            }
-        ]]}, {includes = {"raylib.h"}}))
+        if version:ge("4.5.0") then
+            assert(package:check_cxxsnippets({test = [[
+                void test() {
+                    InitWindow(100, 100, "hello world!");
+                    Camera camera = { 0 };
+                    UpdateCamera(&camera, 0);
+                }
```

Comment:
why? same test code

Reply:
Not same code, UpdateCaemra function in 4.5.0 must provide two arguments.

Reply:
you can remove it, just use InitWindow. or add another api with same arguments to test link.

Reply:
Done, I removed the camera creation